docs/_articles/en/v3.x/scanner-commands/run.md

```diff
@@ -58,7 +58,7 @@ Specifies one or more engines to run. Submit multiple values as a comma-separate
  Throws an error when violations are found with equal or greater severity than the provided value. –normalize-severity is invoked and severity levels are reset to the baseline. Normalized severity values are: 1 (high), 2 (moderate), and 3 (low). Exit code is the most severe violation.
 
  -t, --target=_target_
- (required) Source code location. May use glob patterns. Specify multiple values as a comma-separated list.
+ Specifies the source code location. Use glob patterns or specify individual methods with #-syntax. Multiple values are specified as a comma-separated list. Default is ".".
```

Comment:
This is incorrect. Keep in mind that run dfa is different than run here. No #-syntax support with "run".

This should be:
Specifies the source code location. May use glob patterns. Specify multiple values as a comma-separated list. Default is ".".